### DISCUSSION TITLE
Import exported Shazam .csv files and/or .m3u file to a new playlist.

### DISCUSSION THREAD

**Discussion by UnoNYC** [KRTirtho/spotube#1067] [Ideas]
Hi, I love this app and for the most part, it's been issue free, however, I have a feature suggestion..

It would be nice if we had the ability to import the exported .csv file from Shazam and/or .m3u playlists to the app so that it will create a new playlist of songs if they are on Spotify.

I know that Shazam and Spotify have, or at least, HAD the ability to connect and I have tried this and successfully linked my accounts, but even though they are connected, my Shazams do not show up on Spotify at all, and in Shazam, it only recognizes Apple Music.

Anyway, I also think it would be nice to have the ability to import other playlists so that they will make a new playlist in Spotube so that it'll show up in Spotify much like the other created playlists do already. I think this would be a great option for those of us music lovers who have a bunch of playlists on their devices..

Thank you in advance for your time.

**Comment by KRTirtho** [comment]
Currently, we've no plans to add this feature but now we have this in our record.
Meanwhile, you can use services like [soundiz](https://soundiiz.com/tutorial/shazam-to-spotify)
